### ISSUE TITLE
500 error when fetching submissions

### ISSUE THREAD

**Issue by FrankApiyo** [onaio/kaznet-web#336]
When fetching submissions through the API endpoint: http://127.0.0.1:8000/api/v1/submissions/?, I get a 500 error with the following stack trace: 
```Traceback (most recent call last):
  File "/home/user/.local/share/virtualenvs/kaznet-web-Nz89G9xU/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/home/user/.local/share/virtualenvs/kaznet-web-Nz89G9xU/lib/python3.6/site-packages/django/core/handlers/base.py", line 158, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/user/.local/share/virtualenvs/kaznet-web-Nz89G9xU/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = response.render()
  File "/home/user/.local/share/virtualenvs/kaznet-web-Nz89G9xU/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/home/user/.local/share/virtualenvs/kaznet-web-Nz89G9xU/lib/python3.6/site-packages/rest_framework/response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/home/user/.local/share/virtualenvs/kaznet-web-Nz89G9xU/lib/python3.6/site-packages/rest_framework_json_api/renderers.py", line 660, in render
    render_data, accepted_media_type, renderer_context
  File "/home/user/.local/share/virtualenvs/kaznet-web-Nz89G9xU/lib/python3.6/site-packages/rest_framework/renderers.py", line 104, in render
    allow_nan=not self.strict, separators=separators
  File "/home/user/.local/share/virtualenvs/kaznet-web-Nz89G9xU/lib/python3.6/site-packages/rest_framework/utils/json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
  File "/usr/lib/python3.6/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/usr/lib/python3.6/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.6/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/user/.local/share/virtualenvs/kaznet-web-Nz89G9xU/lib/python3.6/site-packages/rest_framework/utils/encoders.py", line 67, in default
    return super().default(obj)
  File "/usr/lib/python3.6/json/encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'Money' is not JSON serializable

**Comment by FrankApiyo** [comment]
This happens because the amount property of the `Submission` model returns an object of type Money, the amount field of it's associated `Bounty` model here: https://github.com/onaio/kaznet-web/blob/master/kaznet/apps/main/models/submissions.py#L74. The amount field of the Bounty model is a MoneyField.
This will be fixed by creating a custom SerializableMoneyField that returns amount field of the MoneyField instead of the MoneyField itself to API calls.

**Comment by moshthepitt** [comment]
Does a `SerializableMoneyField` already not exist?

**Comment by FrankApiyo** [comment]
No It does not.

**Comment by moshthepitt** [comment]
What about this https://github.com/onaio/kaznet-web/blob/3f0dd142fc49ef0473d468b4131beebefd6e561d/kaznet/apps/main/serializers/bounty.py#L30

**Comment by FrankApiyo** [comment]
This issue was caused by the deletion of this line: https://github.com/onaio/kaznet-web/blob/master/kaznet/apps/main/serializers/submissions.py#L24 in this commit: 2ac24b41c. It has now been fixed by this commit:  5b462a1.
